usaspending_api/etl/management/commands/update_file_c_linkages_in_delta.py

```diff
@@ -150,6 +160,15 @@ def handle(self, *args, **options):
         unlinked_count = self.get_unlinked_count("int")
         logger.info(f"Count of unlinked records after updates: {unlinked_count:,}")
 
+        # Write view back to Postgres for linkages
+        c_to_d_linkage_updates_df = self.spark.sql("SELECT * FROM union_all_priority;")
+        c_to_d_linkage_updates_df.write.jdbc(
+            url=get_usas_jdbc_url(),
+            table="public.c_to_d_linkage_updates",
```

Comment:
Since it's only good for 1 day at a time (i.e. updates since last pipeline run) maybe the `temp.` schema is a better choice here.

Reply:
I was thinking about that too. I ultimately decided to leave it in `public` because the other tables in the `temp` schema are temporary themselves (swapped, dropped, and recreated), whereas only the data itself is temporary in this table. 
I can definitely see the argument though for the `temp` schema though.